gitclone/gitclone.go

```diff
@@ -220,14 +132,7 @@ func setupSparseCheckout(gitCmd git.Git, sparseDirectories []string) error {
 func Execute(cfg Config) error {
 	defer tracker.wait()
 
-	maxEnvLength, err := getMaxEnvLength()
```

Comment:
Moved this inside the functions in `output.go`. The error is still returned, but will be nested in a `export_envs_failed` error (see L213 in the new version). I don't think it's going to cause any issues, this error happens very rarely anyway.